dlna/dms/cds_unix.go

```diff
@@ -0,0 +1,25 @@
+//+build unix
+
+package dms
+
+import (
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+func isHiddenPath(path string) (bool, error) {
+	return strings.Contains(path, "/."), nil
```

Comment:
Will this work for hidden files immediately in the root?

Reply:
Probably not. That's why I was thinking about unit testing.

Reply:
If we knew for certain that path started with /, then this is fine. Static analysis, or another method like always prepending a "/" would work.